packages/model-viewer-effects/src/effects/fxaa.ts

```diff
@@ -0,0 +1,29 @@
+/* @license
+ * Copyright 2023 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FXAAEffect } from 'postprocessing';
+import { MVEffectBase } from './mixins/effect-base.js';
+
+export class MVFXAAEffect extends MVEffectBase {
```

Comment:
Do you want to add this one to your examples? And maybe comment on the pros and cons of MSAA, SMAA, and FXAA?

Reply:
I'm actually considering whether I should include it at all, mainly because the quality is so much worse than the default antialiasing of the WebGLRenderer or of the SMAA effect